README.md

```diff
@@ -255,6 +255,10 @@ CDTQResultSet *result = [datastore find:query];
 
 See [Index and Querying Data](https://github.com/cloudant/CDTDatastore/blob/master/doc/query.md).
 
+For information about migrating your legacy indexing/query code to the new Cloudant Query - iOS implementation.
```

Comment:
These shouldn't be two sentences as they are closely related. I think this needs a more substantial introduction along the lines of "As of version xxx, indexing/query code has been re-written and has many more features than the previous implementation. For implementation about migrating existing indexing /query code see yyy"